source/quickstart/k8s.rst

```diff
@@ -6,35 +6,158 @@ Quickstart for Kubernetes
 
 .. default-domain:: minio
 
-This page deploys a MinIO Tenant onto a Kubernetes cluster for use with early
-development and evaluation environments.
+.. |OS| replace:: Kubernetes
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+This procedure deploys a :ref:`Standalone <minio-installation-comparison>` MinIO server onto |OS| for early development and evaluation of MinIO Object Storage and it's S3-compatible API layer.
```

Comment:
@djwfyi stomped my previous comment.

Looked for a way to further simplify your into paragraph and remove anything we can live without for a quickstart purpose.

Let me know what you think - dropping erasure coding here I think is OK. I'm removing references to "filesystem mode" because with the future single-volume erasure coding, that wont be a "thing" in the same way, so we save ourselves some time now by getting ready for that.